flocker/control/_diffing.py

```diff
@@ -15,6 +17,7 @@
     pvector,
     pvector_field,
 )
+from pyrsistent._transformations import _get
```

Comment:
I don't think this should be public, but instead I've suggested that the ability to perform multiple transformations before checking invariants should be provided by Pyrsistent: https://github.com/tobgu/pyrsistent/issues/89